roles/servicetelemetry/vars/main.yml

```diff
@@ -1,2 +1,45 @@
 ---
 # vars file for servicetelemetry
+servicetelemetry_vars:
+  metrics: |
+    {%- if metrics is defined and metrics is iterable -%}
+    {{ servicetelemetry_defaults.metrics | combine((metrics | stripnone), recursive=True) }}
```

Comment:
This is the magic right here. Super useful!

Reply:
I take no credit for it other than to copy and paste it :)

Reply:
Do you know the blood, sweat, and toil I went through to finally come up with this? I still have the scars. :-D

Reply:
OMG I can only imagine. It's pretty epic though :)

I did some testing on a smaller sample before going through all this work, and it was awesome.

Reply:
> Do you know the blood, sweat, and toil I went through to finally come up with this? I still have the scars. :-D

Thanks!